### PR TITLE
Product detail size changes

### PR DESCRIPTION
**Pull request by kumuthac23**

### CODE REVIEW

File: src/common/component/CommonProductCard.tsx [hunk 1]
```diff
@@ -18,7 +18,11 @@ function CommonProductCard(props: IProps) {
   const theme = useTheme();
 
   return (
-    <Link to={`/detail/${product._id}`} style={{ textDecoration: "none" }}>
+    <Link
+      to={`/detail/${product._id}`}
+      state={{ some: "/diningout" }}
```
Comment (praveensumathi) [inline src/common/component/CommonProductCard.tsx:23]: change the some to "previousPath"

File: src/pages/home/HomePageSlicker.tsx [hunk 2]
```diff
@@ -280,6 +280,7 @@ function HomePageSlicker() {
               renderOption={(props, option) => (
                 <Link
                   to={`/detail/${option._id}`}
+                  state={{ some: "/diningout" }}
```
Comment (praveensumathi) [inline src/pages/home/HomePageSlicker.tsx:283]: change the some to previouspath

File: src/common/component/CommonProductCard.tsx (hunk 1 above)
Comment (kumuthac23) [reply src/common/component/CommonProductCard.tsx]: changed to previousPath

File: src/pages/home/HomePageSlicker.tsx (hunk 2 above)
Comment (kumuthac23) [reply src/pages/home/HomePageSlicker.tsx]: changed to previouspath

**Review by praveensumathi — APPROVED** [review]

**Review by praveensumathi — APPROVED** [review]

**Pull request merged**